### PR TITLE
add license check

### PR DESCRIPTION
**Pull request by steveklabnik**
This will probably fail, let's see what it says! There's [a "fixup" mode](https://github.com/apache/skywalking-eyes#fix-license-headers) if there's a ton of them.

### CODE REVIEW

**Comment by steveklabnik** [comment]
I'll send a PR actually adding the missing headers in the morning 🙏

File: licenserc.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+header:
+  license:
+    spdx-id: MPL-2.0
+
+    content: |
+      This Source Code Form is subject to the terms of the Mozilla Public
+      License, v. 2.0. If a copy of the MPL was not distributed with this
+      file, You can obtain one at https://mozilla.org/MPL/2.0/.
+  paths:
+    - '**/*.rs'
```
Comment (steveklabnik) [inline licenserc.yaml:10]: I am... concerned that this isn't being respected for some reason, given that it's trying to check this file...

**Comment by steveklabnik** [comment]
Okay! This PR should be ready to be merged now.

**Comment by dancrossnyc** [comment]
Thanks, Steve!

**Pull request merged**